modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractTypeScriptClientCodegen.java

```diff
@@ -222,6 +231,49 @@ public String getTypeDeclaration(Property p) {
     }
 
     @Override
+    public String toDefaultValue(Property p) {
+        if (p instanceof StringProperty) {
+            StringProperty sp = (StringProperty) p;
+            if (sp.getDefault() != null) {
+                return "\"" + sp.getDefault() + "\"";
+            }
+            return UNDEFINED_VALUE;
+        } else if (p instanceof BooleanProperty) {
+            return UNDEFINED_VALUE;
```

Comment:
Wouldn't ```false``` be a better default for a boolean property? (or whatever the default in ```BooleanProperty``` is?)

Reply:
In TypeScript, the default is `undefined`, not `false`.